static/templates/user_info_popover_title.handlebars

```diff
@@ -1,2 +1,11 @@
-<div class="popover-avatar" style="background-image: url('{{user_avatar}}');" >
+<div class="popover-avatar" style="background-image: url('{{user_avatar}}');" ></div>
+<div class="popover-gradient"></div>
+{{! Contents of the "message info" popup }}
+<div class="popover_info">
+    <div class="user-full-name" data-dynamic-text>{{user_full_name}}</div>
+    <div class='my_email'>{{user_email}}</div>
+
+    {{#if user_time}}
+    <li>{{ user_time }} {{#tr this}}Local time{{/tr}}</li>
```

Comment:
We should probably remove `<li>` from user_time. Currently it looks like this. Showing the local time of the user is a new feature we merged and for testing it you have to go settings/display settings and choose timezone. Once the timezone is chosen the local time would be shown in the popover of the user account. 
![screenshot from 2017-06-02 22 19 43](https://cloud.githubusercontent.com/assets/7190633/26750897/a00d4f6e-47e1-11e7-8e9b-81e3417db9a1.png)